### ISSUE TITLE
Show CPU usage per thread?

### ISSUE THREAD

**Issue by nikowitt** [psi-probe/psi-probe#550]
Hi there,
is it possible to extend the threads grid to show the CPU time that is used per thread?

Regards,
Niko

**Comment by jander99** [comment]
Something like this?
https://stackoverflow.com/questions/755899/monitor-cpu-usage-per-thread-in-java

**Comment by nikowitt** [comment]
Yes, for instance, so it is possible to directly see which thread is greedy.
